dotCMS/src/main/java/com/dotcms/rest/RoleResource.java

```diff
@@ -56,14 +57,14 @@
 	 * @return
 	 * @throws DotStateException
 	 * @throws DotDataException
-	 * @throws DotSecurityException
 	 * @throws JSONException
 	 */
 
 	@GET
 	@Path("/loadchildren/{params:.*}")
 	@Produces("application/json")
-	public Response loadChildren(@Context HttpServletRequest request, @PathParam("params") String params) throws DotStateException, DotDataException, DotSecurityException, JSONException {
+	public Response loadChildren(@Context HttpServletRequest request, @PathParam("params") String params)
+			throws DotStateException, DotDataException, JSONException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'com.dotmarketing.business.DotStateException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)